sprockets_postgres.py

```diff
@@ -475,13 +477,25 @@ async def _postgres_connect(self) -> bool:
                         DEFAULT_POSTGRES_CONNECTION_TTL)))
         except (psycopg2.OperationalError,
                 psycopg2.Error) as error:  # pragma: nocover
-            LOGGER.warning('Error connecting to PostgreSQL on startup: %s',
-                           error)
+            LOGGER.warning(
+                'Error connecting to PostgreSQL on startup with %s: %s',
+                safe_url, error)
             return False
         self._postgres_connected.set()
         LOGGER.debug('Connected to Postgres')
         return True
 
+    @staticmethod
+    def _obscure_url_password(url):
+        """Generate log safe url with password obscured."""
+        parsed = parse.urlparse(url)
+        if parsed.password:
+            netloc = '{}:*****@{}:{}'.format(parsed.username,
```

Comment:
super tiny nit, but does it make sense to make the masked password the same length as the password? I'm not sure of a best practice here, but the thought is at least you know the password length matches when it's masked. Then again that may be too much information disclosure or not worth the effort.

Reply:
I'm in favor of simplicity here - using a constant string. Arguably safe too.

Reply:
I'm inclined to stick with the constant string for security reasons. I was even wondering if I should partially mask the user name. But I can change if you want.

Reply:
FWIW, I'm in favor of the constant string as well.  Simpler and doesn't expose any information whatsoever.